src/components/ModalFlows/SupportProposal/SupportProposalScreens.js

```diff
@@ -1,39 +1,29 @@
 import React, { useMemo, useState, useCallback } from 'react'
-import PropTypes from 'prop-types'
 import ModalFlowBase from '../ModalFlowBase'
 import SupportProposal from './SupportProposal'
-import useActions from '../../../hooks/useActions'
 
-function SupportProposalScreens({ versionId, id, onDone, stakeProposal }) {
-  const actions = useActions()
+function SupportProposalScreens({ onStakeToProposal, id }) {
```

Comment:
here you can use the `useAction` hook and get the convictionVotingActions as we were doing in the line right bellow